.github/actions/validate-docs-links/package.json

```diff
@@ -25,8 +25,8 @@
     "github-slugger": "1.2.0",
     "gray-matter": "4.0.2",
     "rehype-raw": "4.0.1",
-    "remark-parse": "7.0.1",
-    "remark-rehype": "5.0.0",
+    "remark-parse": "11.0.0",
+    "remark-rehype": "11.1.0",
```

Comment:
I did a local test with both these new versions and they break the pipeline, they have documented breaking changes (remark-rehype v6), so I would suggest avoiding an update at this time @guimachiavelli

Reply:
Sounds like a good idea, yeah. I'm glad I asked your review before blindly merging this one 😅